cmd/agent/install_script.sh

```diff
@@ -254,7 +262,7 @@ if [ -n "$DD_AGENT_MAJOR_VERSION" ]; then
     exit 1;
   fi
   agent_major_version=$DD_AGENT_MAJOR_VERSION
-else
+elif [ "true" == "true" ]; then
```

Comment:
I don't understand this change :thinking:

Reply:
Right, this comes from https://github.com/DataDog/agent-linux-install-script/blob/main/install_script.sh.template#L260. I could probably remove it here, but I didn't want to do any hand modifications to the script after generating it from the template.

Reply:
I will probably remove it here because shellcheck fails on it... And I'll set up shellcheck in the new repo as well :)